src/ast/mod.rs

```diff
@@ -857,28 +857,32 @@ impl fmt::Display for ListAgg {
 
 /// The `ON OVERFLOW` clause of a LISTAGG invocation
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
-pub struct ListAggOnOverflow {
-    pub error: bool,
-    pub filler: Option<Box<Expr>>,
-    pub with_count: bool,
+pub enum ListAggOnOverflow {
+    Error,
```

Comment:
Please add doc-comments for these variants as well: `ON OVERFLOW ERROR` / `TRUNCATE [ <filler> ] WITH[OUT] COUNT`

Reply:
Oops, I misread this. I'll add that now.